code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -320,6 +341,9 @@
 	item_state = "military"
 	max_w_class = WEIGHT_CLASS_SMALL
 	resistance_flags = FIRE_PROOF
+	can_hold = list(
+		/obj/item/ammo_box/magazine/wt550m9
+		)
```

Comment:
This makes the belt ONLY be able to hold wt ammo boxes, is that intended?

Reply:
You have to copy the parent list and add your entry, or add to the list on init (probably not so ideal?)